### DISCUSSION TITLE
Forking my own private repo into my organisation

### DISCUSSION THREAD

**Discussion by alexandercurtis** [community/community#22295] [New to GitHub]
<p>I have a private repo under my own personal GitHub account. I am a member of an Organization, and I would like the organisation to have a fork of my repo that they can work on. I want my repo to remain private to me, and for the Organization’s fork to be accessible within the organization, but private to outsiders.</p>
<p>Simply forking the repo to the organization appears to work, but it appears to be private to me, as my colleagues cannot access it.</p>
<p>Is there a way to grant access to the fork to other members of the Organization?</p>

**Comment by amargautam** [comment, marked as answer]
<p>You can control access on repo <em>settings &gt; Collaborators &amp; teams</em>. You can create/ add teams or add individuals.</p>
<p>Can you check if this is set up for the repo which is in your organization?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e0eca8c16b37bb91874db03f174dbd44078738d4.png" data-download-href="/uploads/short-url/w5LZAyvyVYloEq1U7jrzemG68gA.png?dl=1" title="Screen Shot 2019-03-01 at 7.47.32 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3035438/181070100-f3fa600e-e92a-4d7d-a396-eeb3385f176b.png" alt="Screen Shot 2019-03-01 at 7.47.32 PM.png" data-base62-sha1="w5LZAyvyVYloEq1U7jrzemG68gA" width="690" height="365" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e0eca8c16b37bb91874db03f174dbd44078738d4_2_690x365.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e0eca8c16b37bb91874db03f174dbd44078738d4_2_1035x547.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/e0eca8c16b37bb91874db03f174dbd44078738d4_2_1380x730.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e0eca8c16b37bb91874db03f174dbd44078738d4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-03-01 at 7.47.32 PM.png</span><span class="informations">2112×1120 222 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Reply by hansmbakker** [reply]
I believe that is not an answer to the question - you explained how the OP could grant access to his private repo, but the OP asked how to make a fork in the organization so that his colleagues could work separately?

**Comment by alexandercurtis** [comment]
<p>Thank you - I’m not sure how I missed that.</p>
